service/telemetry/config.go

```diff
@@ -24,6 +24,7 @@ import (
 type Config struct {
 	Logs    LogsConfig    `mapstructure:"logs"`
 	Metrics MetricsConfig `mapstructure:"metrics"`
+	Traces  TracesConfig  `mapstructure:"traces"`
```

Comment:
Do we need the config? can we rely only on the featuregate initially?

Reply:
Hi @bogdandrutu, i think it is necessary unless we are ok enabling just a default propagator once the feature gate is passed. I was instructed to have a configurable option (by you :-) )   in collectors configurations in the last commit, so I added this change. 

let me know if you changed your mind, I can remove this and just support the feature gate + default propagator.

Reply:
I am just a bit worried that we are defining this before otel does, so let's just add a note that this is experimental until otel defines a yaml config.